### ISSUE TITLE
Jerky scroll when used in ListView

### ISSUE THREAD

**Issue by mehmedean** [clockbyte/admobadapter#102]
I am using version 1.4.5 and I integrated adapter to my ListView as defined in the beginning of cookbook. The scrolling gets terrible and jerky on my Nexus 6P with Android 8.1 Oreo. Why is this happening?

I haven't see any ViewHolder pattern for ListView, so is this because of it?

**Comment by kot331107** [comment]
Yeah I guess we should investigate and refactor the code for Advanced ads... I haven't worked on it while Advanced ads were in a beta. Sorry for a late answer
